cache/converter.go

```diff
@@ -109,6 +133,10 @@ func getConverter(ctx context.Context, cs content.Store, desc ocispecs.Descripto
 			return compressorFunc(w, ocispecs.MediaTypeImageLayerGzip)
 		}
 		c.finalize = finalize
+	case compression.Nydus:
+		c.compress = func(w io.Writer) (io.WriteCloser, error) {
+			return nydusify.Convert(ctx, w, nydusify.ConvertOption{})
+		}
```

Comment:
Isn't this unexpected path? It seems that nydus always returns from this function when it reaches at the check of `needsConversion()` or `needsForceCompression()`.

Reply:
It's used to convert others to nydus compression type, for example `compression.Gzip` to `compression.Nydus`.